DasharoPayloadPkg/BlSupportPei/BlSupportPei.c

```diff
@@ -594,7 +600,7 @@ BlPeiEntryPoint (
   //
   LowMemorySize = PldMemInfo.UsableLowMemTop;

```

Comment:
Any use of checking `LowMemorySize` for `0` here?  This doesn't seem likely, but debugging an issue caused by this won't be pleasant.

Reply:
Added an assert to check that it gives enough space instead of just testing against 0. The only possible invalid value is still 0 with the current code, but may be easier to understand by whoever reads it.

https://github.com/Dasharo/edk2/compare/b67eb08260c40f463eca88298e548cc70f8d3831..97938d04f1df2e564c1986fd34dc0dd00467cf54#diff-e11426c5786175fdf345eec361e1c716ab6792d8fddbd5b005e02ab589ad3d7bR605